src/pick.ts

```diff
@@ -0,0 +1,35 @@
+import { Constructor, Target } from '@hippo-oss/dto-decorators';
```

Comment:
Everything else here is a port from `nest-dto`. Changes:

 - Use `Constructor` instead of `Type` to avoid coupling with `@nestjs`
 - Use `Target` instead of `Obj` because it's available in the new definitions.